homeassistant/components/binary_sensor/device_condition.py

```diff
@@ -36,6 +36,10 @@
 CONF_IS_NOT_COLD = "is_not_cold"
 CONF_IS_CONNECTED = "is_connected"
 CONF_IS_NOT_CONNECTED = "is_not_connected"
+CONF_IS_EMPTY = "is_empty"
+CONF_IS_NOT_EMPTY = "is_not_empty"
+CONF_IS_FULL = "is_full"
+CONF_IS_NOT_FULL = "is_not_full"
```

Comment:
What is the definition of "is_not_full" vs. "is_not_empty", they seems like duplicates, and surely will give cause to a lot of confusion.

I am used to having "is_between" which means neither "empty" nor "full".